exp/addons/api/v1alpha4/clusterresourcesetbinding_types.go

```diff
@@ -104,7 +104,7 @@ func (c *ClusterResourceSetBinding) DeleteBinding(clusterResourceSet *ClusterRes
 // +kubebuilder:resource:path=clusterresourcesetbindings,scope=Namespaced,categories=cluster-api
 // +kubebuilder:subresource:status
 // +kubebuilder:storageversion
-
+// +kubebuilder:printcolumn:name="Age",type="date",JSONPath=".metadata.creationTimestamp",description="Time duration since creation of ClusterResourceSetBinding"
```

Comment:
```suggestion
// +kubebuilder:printcolumn:name="Age",type="date",JSONPath=".metadata.creationTimestamp",description="Time duration since creation of ClusterResourceSetBinding"

```

Reply:
Btw is the suggested change just adding a newline? I spent like 5 min looking for a change in line 107 😅

Reply:
Yes it is :) The problem is that otherwise golangci-lint will complain